README.md

```diff
@@ -56,6 +56,16 @@ get tricky, so please leave this directory out of your commits and pull request
 
 The committer doing the final merge will generate the `content/` directory at that time.
 
+### Running using Docker
+
+**Note:** This is an optional method if you don't want to install gems locally and just use a container for development. The helper script is written for UNIX systems but should be fairly easy to modify to get it working on Windows.
+
+Prerequisites: Make sure you have Docker installed on your machine.
+
+For building the site use the command `./run_with_docker.sh server`. This will start the webserver inside a Docker container and port forward to your local machine at `localhost:4000`. We'll use the official Jekyll image to build the site.
+
+For running the tests to verify all links use `./run_with_docker.sh test`.
```

Comment:
For running the website tests, use...

Remove "to verify all links". Tests currently just do that, but soon enough they'll be doing more stuff. So, just abstract away what the tests does.